analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -108,6 +108,13 @@ void Test_IndexerElementAccess()
             //      ^^^^
         }
 
+        void Test_ArrayElementAccess()
```

Comment:
The `Test_` prefix isn't needed, everything is a test 😄 
```suggestion
        void ArrayElementAccess()
```

Reply:
Doens't block merging, just FYI